gemfiles/rails_6.1.gemfile

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in lograge.gemspec
+gemspec path: '..'
+
+group :test do
+  gem 'actionpack', '~> 6.1.0'
+  gem 'activerecord', '~> 6.1.0'
+  # logstash does not release any gems on rubygems, but they have two gemspecs within their repo.
+  # Using the tag is an attempt of having a stable version to test against where we can ensure that
+  # we test against the correct code.
+  gem 'logstash-event', git: 'https://github.com/elastic/logstash', tag: 'v1.5.4'
+  # logstash 1.5.4 is only supported with jrjackson up to  0.2.9
+  gem 'jrjackson', '~> 0.2.9', platforms: :jruby
+  gem 'lines'
```

Comment:
The build looks to be failing with:

```
   LoadError:
       cannot load such file -- thread_safe
```

I believe this should fix it (see also: [`rails_edge.gemfile`](https://github.com/roidrage/lograge/blob/efc4117fd1c2b9afad43e47a3ffaf5d75f8ffbcc/gemfiles/rails_edge.gemfile#L18)):

```suggestion
  gem 'lines'
  gem 'thread_safe'
```